spec/controllers/reports_controller_spec.rb

```diff
@@ -2,6 +2,8 @@
 require "csv"
 
 RSpec.describe ReportsController, type: :controller do
+  render_views
```

Comment:
is this needed? I can run the tests locally without this line

would be nice to know the issue you encountered to document it somewhere

Reply:
@arielj I had to add this because it was failing without it. 

You can see a description over here (in the commit message): https://github.com/fastruby/points/pull/311/commits/71547ee243b983a817d5fd68ed0e373fdd383126

And the failure over here: https://github.com/fastruby/points/actions/runs/6459356901/job/17534960964 -- not really sure how it worked locally for you.